package.json

```diff
@@ -9,7 +9,6 @@
   },
   "scripts": {
     "install-binary": "node scripts/install-binary.js",
-    "pretest": "npm run install-binary",
     "test": "mocha --compilers js:babel-register",
```

Comment:
We could also add a check in `pretest` to see if the binary is installed already, and if not, run this script. What do you think?